### DISCUSSION TITLE
Trouble with running the example extract wisdom command.

### DISCUSSION THREAD

**Discussion by scottrkahler** [danielmiessler/Fabric#599] [General]
`Error: 'NoneType' object has no attribute 'chat'
'NoneType' object has no attribute 'chat'`

The above is the error I get. I can pull down the youtube transcript. Can someone assist?

**Comment by scottrkahler** [comment]
I think I fixed this concern by making sure everything is updated at the start. `sudo apt update' then 'sudo apt upgrade -y'

**Comment by JasinskiSz** [comment]
Same issue here and updating didn't help.

**Comment by emanvidmaker** [comment]
Same issue here updated it and it did not fix; however i think i caused it by removing the youtube api key; using --setup  to space

Edit: By running the `fabric --gui` i discovered that it was an issue detecting ollama 
the console printed `Failed to connect to Ollama. Make sure Ollama is running and accessible` il try to fix it later
